### ISSUE TITLE
Mandatory footnote alert text not accessible to screen readers

### ISSUE THREAD

**Issue by Maren-SSB** [PxTools/pxtools-project#62]
https://statistics-norway.atlassian.net/browse/PXWEB2-808?atlOrigin=eyJpIjoiYzkxYmUxYmRmOTIyNDlhZDllYjU4OTE5YzRkMjE1Y2MiLCJwIjoiaiJ9
